src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
```

Comment:
I would say making `color` variable of type `Color` is beneficial from readability perspective.